app/partials/register-form.cjsx

```diff
@@ -60,6 +72,11 @@ module.exports = createReactClass
     {badNameChars, nameConflict, passwordTooShort, passwordsDontMatch, emailConflict} = @state
 
     <form method="POST" onSubmit={@handleSubmit}>
```

Comment:
I don't see any changes to `handleSubmit` in this PR. Is Panoptes recording that this account is under 16?

Reply:
I don't think so. I believe the only changes are to the frontend content, but we should get a backend confirmation on this before merge.

Reply:
It seems odd to ask this on the registration form, but not actually record which user accounts ticked it.